expat/tests/basic_tests.c

```diff
@@ -159,11 +159,17 @@ START_TEST(test_bom_utf16_le) {
 }
 END_TEST
 
-/* Parse whole buffer at once to exercise a different code path */
 START_TEST(test_nobom_utf16_le) {
   char text[] = " \0<\0e\0/\0>\0";
 
-  if (XML_Parse(g_parser, text, sizeof(text) - 1, XML_TRUE) == XML_STATUS_ERROR)
+  if (g_chunkSize == 1) {
+    // TODO: with just the first byte, we can't tell the difference between
+    // UTF16-LE and UTF8. Avoid the failure for now.
```

Comment:
Would you mind using spelling "UTF-8" and "UTF-16-LE". It looks "a bit too wrong" right now. Thanks!

Reply:
Done.